package.json

```diff
@@ -28,7 +28,7 @@
     "start:example-supply-chain": "yarn build:dev && cd ./examples/supply-chain-app/ && yarn --no-lockfile && yarn start",
     "start:example-carbon-accounting": "CONFIG_FILE=examples/cactus-example-carbon-accounting-backend/example-config.json node examples/cactus-example-carbon-accounting-backend/dist/lib/main/typescript/carbon-accounting-app-cli.js",
     "purge-build-cache": "del-cli .build-cache/*",
-    "clean": "npm run purge-build-cache && del-cli \"./{packages,examples,extensions}/cactus-*/{dist,.nyc_output,src/main/proto/generated/*,src/main/typescript/generated/proto/protoc-gen-ts/*,src/main/typescript/generated/openapi/typescript-axios/*,src/main-server/kotlin/gen/kotlin-spring/src/**/{model,api}/*}\"",
+    "clean": "npm run purge-build-cache && del-cli \"./{packages,examples,extensions}/cactus-*/{dist,.nyc_output,src/main/proto/generated/*,src/main/typescript/generated/proto/protoc-gen-ts/*,src/main/typescript/generated/openapi/typescript-axios/*,src/main-server/kotlin/gen/kotlin-spring/src/**/{model,api}/*}\" && del-cli \"./dist/*\"",
```

Comment:
@outSH `&& del-cli \"./dist/*\"` Will this wipe every directory everywhere recursively that's called `dist`? Could it cause problems with third party dependencies within `node_modules/` that also have `dist` directories after installation (that we don't want to delete)?

Reply:
@petermetz It's meant to clean only top level dist (i.e. `cactus/dist/*`) where `cmd-socketio-server` put's it's build artifacts. I've tested it once again and it did not remove ./dist content in child directories - checked with `./dist` in `./packages/cactus-plugin-ledger-connector-fabric-socketio` and `./packages/cactus-plugin-ledger-connector-sawtooth-socketio`

Reply:
@outSH Understood, thank you very much for double checking!